public/app/core/services/context_srv.ts

```diff
@@ -57,6 +58,14 @@ export class ContextSrv {
     store.set('grafana.sidemenu.pinned', val);
   }
 
+  setActiveState(val) {
+    this.active = val;
+  }
+
+  isActive() {
+    return this.active;
```

Comment:
Thanks for feedback!
The code was a little bit strange, I just googling and copy the code, sorry...
I change the function name, the code becomes enough simple.